komodo/check_up_to_date_pypi.py

```diff
@@ -164,44 +216,28 @@ def main():
             f"{sys.version_info.minor}."
             f"{sys.version_info.micro}"
         ),
-        help="Which python version to upgrade to, defaults to the system python. "
-        "Should provide it on the form: major.minor.micro, though only: major or: "
-        "major.minor is allowed (but might give unexpected results, for example if "
-        "a package requires >3.6, py-version 3.6 will not be considered valid)",
+        help=(
+            "Which python version to upgrade to, defaults to the system python. Should"
+            " provide it on the form: major.minor.micro, though only: major or:"
+            " major.minor is allowed (but might give unexpected results, for example"
+            " if a package requires >3.6, py-version 3.6 will not be considered valid)"
+        ),
     )
 
-    args = parser.parse_args()
-    print(f"Checking against python version: {args.python_version}")
-    yaml = yaml_parser()
-    releases = load_from_file(yaml, args.release_file)
-    repository = load_from_file(yaml, args.repository_file)
+    return parser.parse_args()
 
-    upgrade_proposals = get_upgrade_proposal(
-        releases,
-        repository,
-        args.python_version,
-    )
 
-    if upgrade_proposals:
-        if args.propose_upgrade:
-            insert_upgrade_proposals(upgrade_proposals, repository, releases)
-            print(
-                "Writing upgrade proposals, "
-                "assuming nothing has changed with dependencies..."
-            )
-            with open(args.propose_upgrade, "w") as fout:
-                yaml.dump(releases, fout)
-            write_to_file(repository, args.repository_file)
+def main():
+    args = get_args()
 
-        errors = []
-        for name, versions in upgrade_proposals.items():
-            pypi_latest = versions["suggested"]
-            current_version = versions["previous"]
-            errors.append(
-                f"{name} not at latest pypi version: {pypi_latest}, "
-                f"is at: {current_version}"
-            )
-        sys.exit("\n".join(errors) + "\nFound out of date packages!")
+    print(f"Checking against python version: {args.python_version}")
 
-    else:
-        print("All packages up to date!!!")
+    ReleaseFile()(args.release_file)
```

Comment:
These two lines look odd, I suppose the intention is to do the validation. Is it possible to do something such that this line looks less odd and perhaps includes the string "validate" in some way?

Reply:
.Yes, I can put them in a separate function.